### PR TITLE
Update sequence diagrams and  Add import implementation in Developer guide

### PR DESCRIPTION
**Pull request by Pughal77**
Made the following additions:

1) Created a sequence diagram for import command and referenced it
to another sequence diagram for addCommand in
`ImportSequenceDiagramRef` to simplify main sequence diagram in
`ImportSequenceDiagram`.

2) Updated Developer guide to add implementation of import command.

### CODE REVIEW

File: docs/diagrams/ImportSequenceDiagramRef.puml
Comment (Zer0Legion) [inline docs/diagrams/ImportSequenceDiagramRef.puml:1]: should sd encapsulate the entire box? something like this in the textbook:
image

**Review by jayllo-c — COMMENTED** [review]
Great job with the documentation of the implementation details. It is both detailed and clear.

File: docs/diagrams/ImportSequenceDiagramRef.puml
Comment (delishad21) [reply docs/diagrams/ImportSequenceDiagramRef.puml:1]: Agree with lengkhai on this, the formatting here seems a little iffy.

Comment (delishad21) [reply docs/diagrams/ImportSequenceDiagramRef.puml:1]: Agree w lengkhai on this, the formatting seems a little iffy

**Review by delishad21 — COMMENTED** [review]
Not too sure if this formatting is correct. Maybe you can post on the forum to check?

**Pull request merged**